src/components/ui/combobox.tsx

```diff
@@ -0,0 +1,256 @@
+'use client'
+
+import * as React from 'react'
+import { type LucideIcon } from 'lucide-react'
+import {
+  Command,
+  CommandEmpty,
+  CommandGroup,
+  CommandInput,
+  CommandItem,
+  CommandList,
+  CommandShortcut
+} from '@/components/ui/command'
+import {
+  Popover,
+  PopoverContent,
+  PopoverTrigger
+} from '@/components/ui/popover'
+import {
+  Drawer,
+  DrawerContent,
+  DrawerTrigger
+} from '@/components/ui/drawer'
+import { Square, SquareCheck } from '@/components/icons'
+import { useMediaQuery } from '@/hooks/useMediaQuery'
+import { cn } from '@/lib/utils'
+import { Button } from './button'
+
+interface DefaultValueOption {
+  payload?: unknown
+  label: string
+  value: string
+  icon?: LucideIcon
+  avatar?: React.ReactNode
+  iconProps?: {
+    size?: number
+    fill?: string
+    color?: string
+    strokeWidth?: number
+    className?: string
+  }
+  color?: string
+  info?: string
+}
```

Comment:
Not sure where this type belongs, rather not keep it duplicated though. Which would probably mean that it should reside here in `elephant-ui`, be exported, and removed from `elephant-chrome`.

Reply:
fixed in new commit